website/src/static/markdown/showcase.md

```diff
@@ -0,0 +1,61 @@
+<div>
+  <div class="thumb">
+    <div class="bg-black" data-title="Kepler.gl" >
+      <a href="https://uber.github.io/kepler.gl/">
+        <img src="images/showcase-kepler-gl.jpg" />
+      </a>
+    </div>
+  </div>
+  <div class="thumb-desc">
+    <div>
+      <h2>Kepler.gl</h2>
+      <p>Kepler.gl is a powerful web-based geospatial data analysis tool. Built on a high performance rendering engine and designed for large-scale data sets.</p>
+      <p class="credit">By <a href="http://vis.gl/about">Uber Visualization</a> | <a href="">GitHub</a><p>
+    </div>
+  </div>
+
+  <div class="thumb">
+    <div class="bg-black" data-title="Wind Map">
+      <a href="http://uber.github.io/deck.gl/showcase/wind">
+        <img src="images/showcase-wind.jpg" />
+      </a>
+    </div>
+  </div>
+  <div class="thumb-desc">
+    <div>
+      <h2>Wind Map</h2>
+      <p>GPU-based data interpolation and partical effects.</p>
+      <p class="credit">By <a href="http://vis.gl/about">Uber Visualization</a> | <a href="https://github.com/uber/deck.gl/tree/5.2-release/examples/wind">Source code</a> | <a href="https://medium.com/vis-gl/wind-map-a58575f87fe3">Blog post</a></p>
+    </div>
+  </div>
+
+  <div class="thumb">
+    <div data-title="Ascii Video Player">
+      <a href="http://pessimistress.github.io/ascii/">
+        <img src="images/showcase-ascii.jpg" />
+      </a>
+    </div>
+  </div>
+  <div class="thumb-desc">
+    <div>
+      <h2>Ascii Video Player</h2>
+      <p>Video processing on the fly.</p>
+      <p class="credit">By <a href="http://vis.gl/about">Uber Visualization</a> | <a href="https://github.com/pessimistress/deck.gl/tree/master/examples/ascii">Source code</a></p>
```

Comment:
broken link if we were not to merge it to master.
href="https://github.com/Pessimistress/deck.gl/tree/ascii/examples/ascii"

Reply:
I have fixed the cross-origin issue, will open a PR to merge this demo into the main repo.